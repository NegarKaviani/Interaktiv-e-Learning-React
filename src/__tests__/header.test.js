import { render, screen } from '@testing-library/react'
import Header from '../components/header';

test("Example 1 renders successfully", () => {
    render(<Header/>);

    const element = screen.getByAltText('Logo');

    expect(element).toBeInTheDocument();
})