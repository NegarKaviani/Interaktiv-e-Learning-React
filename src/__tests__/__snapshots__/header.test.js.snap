// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="logo-line bg-sky-700 h-16 flex flex-row-reverse px-20"
>
  <img
    alt="Logo"
    className="w-auto h-full object-contain"
    src="logo.png"
  />
</div>
`;
